pom.xml

```diff
@@ -53,7 +54,7 @@
         <version.httpcomponents-httpclient>4.5.5</version.httpcomponents-httpclient>
         <version.httpcomponents-httpcore>4.4.8</version.httpcomponents-httpcore>
         <version.infinispan>8.2.4.Final</version.infinispan>
-        <version.jackson>2.7.4</version.jackson>
+        <version.jackson>2.9.6</version.jackson>
```

Comment:
I don't think we can change this version. I believe we include jackson as a provided dependency in order for the code to compile. However, at runtime, the version actually comes from Wildfly and Wildfly 10.1.0.Final (which is what we build/test/run against) uses Jackson 2.7.4. Even if this change doesn't break anything, it's misleading since we won't actually be using the newer version at runtime.

Reply:
According to: https://github.com/wildfly/wildfly/blob/10.1.0.Final/pom.xml you are correct. These other libraries are also provided:
<version.commons-beanutils>1.9.2</version.commons-beanutils> <-- I'll change pull request to match this version
<version.commons-cli>1.2</version.commons-cli>   <--- provided by Wildfly 10.1.0.Final - but doesn't match datawave pom.xml, Should we correct??
<version.commons-codec>1.10</version.commons-codec>
<version.commons-collections>3.2.2</version.commons-collections>
<version.commons-pool>1.6</version.commons-pool>
<version.org.infinispan>8.2.4.Final</version.org.infinispan>
<version.org.javassist>3.18.1-GA</version.org.javassist> <-- doesn't match version in dwv pom.xml
And possibly others I missed.
<version.jgroups>3.6.10.Final</version.jgroups>
<version.org.slf4j>1.7.7.jbossorg-1</version.org.slf4j>  <-- doesn't match version in dwv pom.xml

Maybe we should organize the imports in the datawave pom.xml that are actually from Wildfly into a specific section with comments that they need to match the version that is in the Wildfly version being used? Or pull them together and marked them as provided?

Reply:
I would say in general the dependencies need a review. Part of the problem is that some code is used in Wildfly, which means for certain dependencies we want them to be provided and their versions should probably match that which is actually provided by Wildfly. Others are used in parts of the code that are not run in Wildfly (e.g., ingest and MapReduce code, Accumulo iterators, etc.). The same library (slf4j, for example) may be referenced in multiple parts of the code but only marked as provided when used in modules that we know will run in Wildfly. It's kind of a mess right now.

Reply:
Bumping to this version also breaks many tests.